client/core/core.go

```diff
@@ -943,6 +943,44 @@ func (c *Core) Login(pw []byte) ([]*db.Notification, error) {
 	return notes, nil
 }
 
+// Logout logs the user out
+func (c *Core) Logout() error {
+	c.connMtx.RLock()
+	defer c.connMtx.RUnlock()
```

Comment:
This prevents `c.conns` from being modified, but it does not prevent another goroutine from running a method like `(*Core).Trade` concurrently.  In the case of `(*Core).Trade` it's a race from  `dc.hasOrders(assetID)` to `dc.acct.lock()` while `Trade` could be inserting a new trade.  Using `c.connMtx.Lock()/Unlock()` should prevent this, but please check my reasoning.

Reply:
Yes, only one goroutine rw at the time logout